### PR TITLE
DTSPB-699: Investigate implementing Open ID Connect (OICD) - Reducing…

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-699

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Comment by srochani** [comment]
Closing the PR

**Comment by srochani** [comment]
opening the PR

**Review by nigeldunne — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/fee/SolCcdServiceFeeTests.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 public class SolCcdServiceFeeTests extends IntegrationTestBase {
 
     private static final int APP_FEE = 27300; //comment this out for local tests - keep for commits
-    //private static final int APP_FEE = 15500;
+    // private static final int APP_FEE = 15500;
```
Comment (isha2040) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/fee/SolCcdServiceFeeTests.java:30]: can we delete this commented code

Comment (srochani) [reply src/functionalTest/java/uk/gov/hmcts/probate/functional/fee/SolCcdServiceFeeTests.java:30]: It meant to be keep, as for local env we have to change to 15500 as per Sanjay's change

**Review by isha2040 — APPROVED** [review]

**Pull request merged**